elastic/kibana
# [Infra UI] Refactor host view and asset details dashboards configuraton

closes [#163797](https://github.com/elastic/kibana/issues/163797)

## Summary

This PR extracts the dashboard configuration from the components that renders them to common. This aims to make discoverability and maintainability easier.

It doesn't change any functional behaviour


### How to test

- Start a local kibana instance
- Navigate to `Infrastructure` > `hosts`
- Verify if the charts still work as expected
